tucan/io.py

```diff
@@ -24,10 +24,21 @@ def graph_from_file(filepath: str) -> nx.Graph:
     filepath = Path(filepath)
     if filepath.suffix != ".mol":
         raise IOError(f"The file must be in '.mol' format, not {filepath.suffix}.")
-    filecontent = _read_file(filepath)
+    with open(filepath) as file:
+        filecontent = file.read()
+
+    return graph_from_molblock(filecontent)
+
+
+def _split_into_tokenized_lines(string: str) -> List[List[str]]:
```

Comment:
I like that the tokenization is factored out now. However, could we keep `_read_file()` and have it return the result of `split_into_tokenized_lines()`?  Then we nicely encapsulate 1. reading the file, 2. tokenizing the file.  Also makes testing easier, the mocked `_read_file()` in the IO tests makes me slightly uncomfortable 🙈

Reply:
Ok, fixed. I usually have some kind of testutils module/class in my projects that takes care for reading test resource files. ;)